app/src/main/java/com/duckduckgo/app/statistics/VariantManager.kt

```diff
@@ -33,18 +33,18 @@ interface VariantManager {
     companion object {
 
         // this will be returned when there are no other active experiments
-        val DEFAULT_VARIANT = Variant(key = "", features = emptyList())
+        val DEFAULT_VARIANT = Variant(key = "", features = emptyList(), filterBy = { true })
 
         val ACTIVE_VARIANTS = listOf(
 
             // SERP variants. "sc" may also be used as a shared control for mobile experiments in
             // the future if we can filter by app version
-            Variant(key = "sc", weight = 0.0, features = emptyList()),
-            Variant(key = "se", weight = 0.0, features = emptyList()),
+            Variant(key = "sc", weight = 0.0, features = emptyList(), filterBy = { true }),
```

Comment:
Let's use a common filter lambda for clarity and readability, as discussed in the tech design:
`Variant(key = "sc", weight = 0.0, features = emptyList(), filterBy = { noFilter() }),`

Reply:
Done!